examples/ember-cli/package.json

```diff
@@ -5,12 +5,13 @@
   "scripts": {
     "build": "ember build --output-path ember-output",
     "build-storybook": "yarn storybook-prebuild && build-storybook -s ember-output",
-    "dev": "ember serve",
+    "dev": "ember serve --output-path ember-output",
     "storybook": "yarn build && start-storybook -p 9009 -s ember-output",
     "storybook-prebuild": "yarn build && shx cp -r public/* ember-output",
-    "storybook:dev": "yarn dev & start-storybook -p 9009 -s ember-output"
+    "storybook:dev": "echo 'run `yarn dev` in another terminal' && start-storybook -p 9009 -s ember-output"
```

Comment:
Should be first ember to start then running `start-storybook` in another terminal WDYT? To avoid concurrency issues btw ember not ready and storybook consuming ember output